### DISCUSSION TITLE
root permission

### DISCUSSION THREAD

**Discussion by whale-fall-0803** [lima-vm/lima#1484] [Q&A] [answered]
### Description

I have a problem. When I run the Centos virtual machine. I can use the limactl shell bash to enter the command line. But if I need to make a software installation request. So I need to obtain root permission, so when I perform sudo operations. How can I know my root password?

**Comment by lobshunter** [comment, marked as answer]
It should be viable to change password with `sudo passwd`, or simply run commands as root by `sudo $COMMAND`

**Comment by whale-fall-0803** [comment]
OK. Thank you very much. I'll give it a try



| |
whale fall
|
|
***@***.***
|


---- Replied Message ----
| From | ***@***.***> |
| Date | 04/14/2023 11:09 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | Re: [lima-vm/lima] root permission (Issue #1483) |

It should be viable to change password with sudo passwd, or simply run commands as root by sudo $COMMAND
